.github/workflows/zxcron-extended-test-suite.yaml

```diff
@@ -327,8 +333,22 @@ jobs:
                       "type": "section",
                       "text": {
                         "type": "mrkdwn",
-                        "text": "*Source Commit*: \n<${{ github.server_url }}/${{ github.repository }}/commit/${{ github.sha }}>"
-                      }
+                        "text": "*Source Commit*: \n<${{ github.server_url }}/${{ github.repository }}/commit/${{ needs.fetch-xts-candidate.outputs.xts-tag-commit }}>"
+                      },
+                      fields: [
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Workflow run ID*: ${{ github.run_id }}"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Workflow run URL*: \n<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}>"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Commit author*: ${{ needs.fetch-xts-candidate.outputs.xts-tag-commit-author }}"
+                        }
+                      ]
```

Comment:
I like this approach. I would recommend updating the fields to so that they're a little more tabular, though.

```
"type": "section",
			"text": {
				"type": "mrkdwn",
				"text": "*Workflow and Commit Information*"
			},
			"fields": [
				{
					"type": "mrkdwn",
					"text": "*Source Commit*:"
				},
				{
					"type": "mrkdwn",
					"text": "<${{ github.server_url }}/${{ github.repository }}/commit/${{ needs.fetch-xts-candidate.outputs.xts-tag-commit }}>"
				},
				{
					"type": "mrkdwn",
					"text": "*Commit author*:"
				},
				{
					"type": "mrkdwn",
					"text": "${{ needs.fetch-xts-candidate.outputs.xts-tag-commit-author }}"
				},
				{
					"type": "mrkdwn",
					"text": "*Workflow run ID*:"
				},
				{
					"type": "mrkdwn",
					"text": " ${{ github.run_id }}"
				},
				{
					"type": "mrkdwn",
					"text": "*Workflow run URL*:"
				},
				{
					"type": "mrkdwn",
					"text": "<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}>"
				}
			]
```

You can use the slack block kit builder to play with the formatting: https://app.slack.com/block-kit-builder/

Reply:
I've added some tabulation, but not too much. At least for me, it seems like a bigger mess with too much tabs.

Reply:
At the very least I recommend re-ordering the items as I suggested above. That way we see commit information and then workflow information. 

The way that the `fields` type works is that each field represents an entry in a 2 column table. Right now if this were to hit the slack channel it would be a bit messy to read; however doing the output similar to how i've suggested it (and similar to how the first `field` section is implemented) will clean up the view that the end-user will see.

Reply:
Sorry, I've completely misunderstood what your point was. I hope that the formatting is ok now.